istio-control/istio-config/templates/deployment.yaml

```diff
@@ -66,6 +66,7 @@ spec:
           - --validation-webhook-config-file
           - /etc/config/validatingwebhookconfiguration.yaml
           - --monitoringPort=15014
+          - --validation-port=9443
```

Comment:
Does the flag exists in 1.2 ? Upgrade.

Reply:
This flag is just temporarily fix in installer. it should be not required when this PR is merged - https://github.com/istio/istio/pull/16015. You can see that there does not get the default value wile just hardcoded.
https://github.com/istio/istio/blob/8757a56331a384bae999dbdbb165eea6175ffd60/galley/cmd/galley/cmd/server.go#L160